src/troute-network/troute/AbstractRouting.py

```diff
@@ -74,8 +72,10 @@ def read_parquet(file_path):
     
 
     '''
-    tbl = pq.read_table(file_path)
-    df = tbl.to_pandas().dropna(subset=['hy_id','relative_dist','Z','roughness'])
+    df = pd.read_parquet(file_path,
```

Comment:
When I use the newest topodata with manning's value attached, I have this error:
![image](https://github.com/NOAA-OWP/t-route/assets/54687070/77812858-f2fc-498e-ba86-7d699cfddfae)
It seems that the column names of the new topo data is different from the previous version: For example, here:
https://github.com/kumdonoaa/t-route/blob/ee4be4b549625392229cdff6a4e0fea7ce0026cb/src/troute-network/troute/AbstractRouting.py#L78

Reply:
Yeah, I preprocessed the raw topobathy parquet files to create a `roughness` column that is the average of `min_n` and `max_n`. Do you think we should just use `min_n`?

Reply:
Oh, this is actually an error about `relative_distance`. That was also preprocessed, changed from `relative_distance` to `relative_dist`.

Reply:
The problem is troute_test.parquet has different column names than the newest DEM parquet files. So we either need consistency across parquet files, or a more complicated method of reading files with different formats.

Reply:
I think averaged manning's n may be ok for this functional test but I need to study more how min_n and max_n have been developed.  In the code I attached in the last line of my previous comment, I simply picked min_n to represent n.  I think we may have crosswalk table or something to filter different names and converge into common name.